packages/styleguide/styleguide.root.js

```diff
@@ -98,6 +98,7 @@ Object.assign(Vue.prototype, {
     return '/';
   },
   $route: { query: {} },
+  localePath: (args) => args,
```

Comment:
flagged as duplicate, remove the one at the top and have this fallback to `/` if that's needed still.